client/src/components/Image.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Picture = styled.img`
+  /* height: 50%; */
+  max-width: 100%;
+  max-height: 100%;
+  padding: 15px;
+  border-radius: 20px;
+`
+const Video = styled.video`
+  /* height: 50%; */
+  max-width: 100%;
+  max-height: 100%;
+  padding: 15px;
+  border-radius: 20px;
+`
+const Wrapper = styled.div`
+  /* display: flex; */
+  width: 100%;
+  overflow: hidden;
+`
+
+class Image extends React.Component {
```

Comment:
@gigicupp does this need to be a class based component? If you prefer it, thats fine but I think when a component doesnt explicitly manage its own state the best practice is to use a functional stateless component. I am not positive about that though, something to think about if you have time in the future!

Reply:
@masoncpott i may add another child component. so I always start with class and refactor to functional if not needed.